go/libkb/login_state.go

```diff
@@ -1006,3 +1033,14 @@ func (s *LoginState) AccountDump() {
 		s.G().Log.Warning("error getting account for AccountDump: %s", err)
 	}
 }
+
+// WaitForTimerToComplete. Only currently used in testing. Returns a function
```

Comment:
Still thinking about alternate ways to do this, but in the meantime, this should be GetTimerCompletionWaiter.